scripts/windows/deploy.one.cmd

```diff
@@ -34,8 +34,6 @@ REM
     set rdst=\\%machine%\%rdst_dir%
     @mkdir %rdst%    
     xcopy /F /Y /S %src_dir% %rdst%
-    COPY /Y %bin_dir%\7z.exe %rdst%
```

Comment:
there is chick-and-egg issue. The monitor's pack needs 7z to unzip itself, after that the monitor pack may install 7z itself (i'm not sure the later stuff though)